index.html

```diff
@@ -2,14 +2,37 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css"
+          href="https://fonts.googleapis.com/css?family=Martel+script">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js" charset="utf-8"></script>
     <title>
     </title>
   </head>
+  <header>
+    <div class="navbar">
+      <p id="busmall">BusMall</p>
```

Comment:
If this is supposed to be a special block of text, why not use an `<h*>` tag instead of a `<p>` tag?